lona/html/data_binding/inputs.py

```diff
@@ -1,101 +1,35 @@
-from copy import copy
-
-from lona.html.nodes import TextAreaNode, InputNode, Widget
 from lona.events.event_types import CHANGE
+from lona.html.node import Node
 
 DEFAULT_INPUT_DELAY = 300
 
 
-class TextInput(Widget):
-    NODE_CLASS = InputNode
-
-    ID_LIST = []
-    CLASS_LIST = []
-    STYLE = {}
+class TextInput(Node):
+    TAG_NAME = 'input'
+    SELF_CLOSING_TAG = True
 
     ATTRIBUTES = {
         'type': 'text',
     }
 
-    def __init__(self, value=None, disabled=False, bubble_up=False,
-                 input_delay=DEFAULT_INPUT_DELAY, **input_kwargs):
-
-        self.bubble_up = bubble_up
-
-        self.input_node = self.gen_input_node(**input_kwargs)
+    def __init__(self, *args, value=None, disabled=False, bubble_up=False,
```

Comment:
Input is self closing and can't contain children.
```suggestion
    def __init__(self, value=None, disabled=False, bubble_up=False,
```

Reply:
Done